tools/prow-config-updater/config/config.go

```diff
@@ -110,7 +110,14 @@ func UpdateProw(env ProwEnv, dryrun bool) error {
 	return helpers.Run(
 		fmt.Sprintf("Updating Prow configs with command %q", updateCommand),
 		func() error {
-			out, err := cmd.RunCommand(updateCommand)
+			kf := os.Getenv("GOOGLE_APPLICATION_CREDENTIALS")
+			if kf != "" {
```

Comment:
I think this deserves a comment, if this is not set then we assume it's local?

Reply:
Added the comment. If the env var is not set, it will fallback to use the local credential.